ca_gazebo/launch/apriltag.launch

```diff
@@ -0,0 +1,34 @@
+<launch>
```

Comment:
Maintain the naming convention for launch files.

Reply:
Adressed